services/director-v2/src/simcore_service_director_v2/modules/resource_usage_client.py

```diff
@@ -0,0 +1,137 @@
+""" Interface to communicate with the resource usage tracker
+"""
+
+import contextlib
+import logging
+from dataclasses import dataclass
+from typing import cast
+
+import httpx
+from fastapi import FastAPI
+from models_library.api_schemas_webserver.resource_usage import PricingPlanGet
+from models_library.products import ProductName
+from models_library.resource_tracker import PricingDetailId, PricingPlanId
+from models_library.services import ServiceKey, ServiceVersion
+from pydantic import parse_obj_as
+
+from ..core.settings import AppSettings
+
+_logger = logging.getLogger(__name__)
+
+
+@dataclass
+class ResourceUsageApi:
+    client: httpx.AsyncClient
+    exit_stack: contextlib.AsyncExitStack
+
+    @classmethod
+    def create(cls, settings: AppSettings) -> "ResourceUsageApi":
+        client = httpx.AsyncClient(
+            base_url=settings.DIRECTOR_V2_RESOURCE_USAGE_TRACKER.api_base_url,
+        )
```

Comment:
I think we have a shared http client for some requests. Do we want to create a new one here?
@pcrespov @sanderegg

Reply:
For now I will create new one and we will sync with @pcrespov how to build some base class